### ISSUE TITLE
Wrong message "Latte template was not set for SomeControl::render"

### ISSUE THREAD

**Issue by lulco** [efabrica-team/phpstan-latte#271]
Code in php is:
```
$this->template->setFile(__DIR__ . '/templates/' . $this->templateFileName . '.latte');
```

I would expect error like "Latte template cannot be resolved"

**Comment by MartinMystikJonas** [comment]
Can you show entire method? MIght be solvet by #273

**Comment by lulco** [comment]
Entire method
```
private string $templateFileName;

public function render(): void
{
    $this->template->setFile(__DIR__ . '/templates/' . $this->templateFileName . '.latte');
    // some variables assigning
    $this->template->render();
}
```

Yes I think it is solved by #273, message is much better. Although, I have two errors instead of one, but at least they are at the right place :)
```
  44     Cannot automatically resolve latte template from expression.  
  48     Cannot resolve rendered latte template.                       
```

Fixed by adding typehint (names of templates) to `private string $templateFileName;` 

```
/** @var "logged"|"unlogged" */
private string $templateFileName;
```
